test/python/cross-grpc-version/pool/features/pool.feature

```diff
@@ -8,6 +8,11 @@ Feature: Mayastor pool management
     When the user creates a v1 version pool with the name of an existing pool
     Then the pool create command should fail
 
+  Scenario: creating a v1 version pool with same name and different disk of existing v0 version pool
+    Given a v0 version pool "p2" on "disk3"
+    When the user creates a v1 version pool with the same name and different disk of an existing pool
+    Then the pool create request should fail
```

Comment:
should we specify what kind of grpc error we get?

Reply:
It was not mentioned in other bdd written in mayastor repo. Hence decided to stick to convention.